src/lightning_app/cli/cmd_clusters.py

```diff
@@ -25,10 +28,26 @@
 from lightning_app.utilities.network import LightningClient
 from lightning_app.utilities.openapi import create_openapi_object, string2dict
 
-CLUSTER_STATE_CHECKING_TIMEOUT = 60
 MAX_CLUSTER_WAIT_TIME = 5400
 
 
+class ClusterState(Enum):
```

Comment:
@carmocca maybe we can consider to move StrEnum to utils so we can use it also elsewhere :)

Reply:
It's already there: https://github.com/Lightning-AI/utilities/blob/cdedd59bf02a9035bf8828dea263767c934f55b8/src/lightning_utilities/core/enums.py#L9

Reply:
I'll use that instead of `Enum`